app/assets/javascripts/sufia_overrides.js

```diff
@@ -0,0 +1,6 @@
+// These variables override those defined in Sufia's uploader.js.
+// application.js makes sure that this file is loaded after Sufia.
+var max_file_size = 4000000000;
```

Comment:
Identifier 'max_file_size' is not in camel case.<br>'max_file_size' is defined but never used.